tests/sentry/api/endpoints/test_organization_traces.py

```diff
@@ -496,6 +498,7 @@ def test_use_first_span_for_name(self):
                 "project": self.project.slug,
                 "start": timestamp,
                 "trace": trace_id,
+                "rootDuration": 60100,
```

Comment:
nit, should we format these all the same:
```suggestion
                "rootDuration": 60_100,
```